specs/meta/validator-deposit.md

```diff
@@ -0,0 +1,78 @@
+<!-- DOCTOC SKIP -->
+
+# Validator Deposit Guide
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Deposit into `ValidatorPool`](#deposit-into-validatorpool)
+- [Withdraw from `ValidatorPool`](#withdraw-from-validatorpool)
+- [Try unbond in `ValidatorPool`](#try-unbond-in-validatorpool)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+We want the [validator](../../components/validator/) role to be decentralized. Like how the PoS mechanism works, to
+achieve this, the validator needs to bond ETH at every
+[output submission](../validations.md#submitting-l2-output-commitments). A Validator can deposit into the contract
+`ValidatorPool` at once for
+convenience. This guide teaches you how to deposit, withdraw, and try to unbond in `ValidatorPool`.
+
+## Deposit into `ValidatorPool`
+
+```shell
+> cd components/validator
+```
+
+```shell
+> go run ./cmd/main.go \
+> --valpool-address <validator-pool-address> \ # must be set
+> --l1-eth-rpc <l1-eth-rpc> \
+> --mnemonic <mnemonic> \
+> --hd-path <hd-path> \
+> --rollup-rpc "" \ # empty required flags
+> --l2oo-address "" \
+> --colosseum-address "" \
+> --challenger.poll-interval 0s \
+> balance deposit \
+> --deposit-amount <deposit-amount-wei> # must be set
+```
+
+## Withdraw from `ValidatorPool`
+
+```shell
+> cd components/validator
+```
+
+```shell
+> go run ./cmd/main.go \
+> --valpool-address <validator-pool-address> \ # must be set
+> --l1-eth-rpc <l1-eth-rpc> \
+> --mnemonic <mnemonic> \
+> --hd-path <hd-path> \
+> --rollup-rpc "" \ # empty required flags
+> --l2oo-address "" \
+> --colosseum-address "" \
+> --challenger.poll-interval 0s \
+> balance withdraw \
+> --withdraw-amount <withdraw-amount-wei> # must be set
+```
+
+## Try unbond in `ValidatorPool`
+
+```shell
+> cd components/validator
+```
+
+```shell
+> go run ./cmd/main.go \
+> --valpool-address <validator-pool-address> \ # must be set
+> --l1-eth-rpc <l1-eth-rpc> \
+> --mnemonic <mnemonic> \
+> --hd-path <hd-path> \
+> --rollup-rpc "" \ # empty required flags
+> --l2oo-address "" \
+> --colosseum-address "" \
+> --challenger.poll-interval 0s \
+> balance unbond
+```
```

Comment:
```suggestion
```shell
> go run ./components/validator/cmd/main.go \
  --valpool-address <validator-pool-address> \ # must be set
  --l1-eth-rpc <l1-eth-rpc> \
  --mnemonic <mnemonic> \
  --hd-path <hd-path> \
  --rollup-rpc "" \ # empty required flags
  --l2oo-address "" \
  --colosseum-address "" \
  --challenger.poll-interval 0s \
  balance unbond
```
```